examples/tts/conf/hifigan/model/generator/v4.yaml

```diff
@@ -0,0 +1,8 @@
+# @package _group_
```

Comment:
Isn't this file the same as v1?

Reply:
upsampling rates here are [8,8,4,2] vs [8,8,2,2] of v1. In the ADLR checkpoints it was [8,8,4,2], so I created a new one to keep it consistent. But we can just remove it and use v1..

Reply:
got it